### PR TITLE
Update analytics.md for AWS Kinesis Firehose Provider

### PR DESCRIPTION
**Pull request by seang96**
Description of changes:
See PR#[2823](https://github.com/aws-amplify/amplify-js/pull/2823) for the new provider. This PR adds documentation for it.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: js/analytics.md [hunk 1]
```diff
@@ -337,6 +336,58 @@ Analytics.record({
...
+        }
+    ]
+}
+```
+
+For more information visit [Amazon Kinesis Developer Documentation](https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html).
+
+Configure Kinesis Firehose:
+
+Kinesis Firehose uses the same configuration options as the AWS Kinesis Provider. To see how to configure, please visit [Amazon Kinesis Installation & Configuration](https://aws-amplify.github.io/docs/js/analytics#installation-and-configuration).
```
[the remark this answers is not in the document]
Comment (seang96) [reply js/analytics.md]: Ah nice catch, I did forget about that.

[the remark this answers is not in the document]
Comment (seang96) [reply js/analytics.md]: My newest commit has this clarified

**Comment by powerful23** [comment]
@seang96 can you fix the tests for this pr? https://github.com/aws-amplify/docs#contributing

**Comment by mlabieniec** [comment]
@seang96 the doc just needs an update to the spelling exclusion file (see the circleci log)

**Comment by iartemiev** [comment]
Closing this, have created a copy with fixed spelling errors here: https://github.com/aws-amplify/docs/pull/1021

**Pull request closed without merging**